pallets/operation/src/lib.rs

```diff
@@ -218,6 +220,11 @@ pub mod pallet {
     #[pallet::storage]
     pub(super) type StorageVersion<T> = StorageValue<_, Releases>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn npow_day_minted_dpr)]
+    pub type NpowDayMintedDPR<T: Config> =
+        StorageMap<_, Blake2_128Concat, EraIndex, BalanceOf<T>, OptionQuery>;
```

Comment:
use StorageValue<(EraIndex,Balanceof<T>)> is enough for calculating current era limit.
if using StorageMap , need clear old era's record

Reply:
Good point, changed to storagevalue now.